pkg/provisioner/dependency.go

```diff
@@ -125,6 +135,8 @@ func (d *DependencyResolver) withKubernetes() {
 		d.Dependencies = append(d.Dependencies, functions[kubeadmInstaller])
 	case kindInstaller:
 		d.Dependencies = append(d.Dependencies, functions[kindInstaller])
+	case nvkindInstaller:
+		d.Dependencies = append(d.Dependencies, functions[nvkindInstaller])
```

Comment:
It's strange that we have a dependency on kind for `nvkind`, but this is not defined here and is defined at a template level instead.